bundles/aliyun-bundle/src/main/java/org/apache/gravitino/oss/fs/OSSFileSystemProvider.java

```diff
@@ -18,23 +18,41 @@
  */
 package org.apache.gravitino.oss.fs;
 
+import com.google.common.collect.ImmutableMap;
 import java.io.IOException;
 import java.util.Map;
 import org.apache.gravitino.catalog.hadoop.fs.FileSystemProvider;
+import org.apache.gravitino.storage.OSSProperties;
 import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.fs.FileSystem;
 import org.apache.hadoop.fs.Path;
 import org.apache.hadoop.fs.aliyun.oss.AliyunOSSFileSystem;
+import org.apache.hadoop.fs.aliyun.oss.Constants;
 
 public class OSSFileSystemProvider implements FileSystemProvider {
+
+  private static final String OSS_FILESYSTEM_IMPL = "fs.oss.impl";
+
+  public static final Map<String, String> GRAVITINO_KEY_TO_OSS_HADOOP_KEY =
```

Comment:
Why do we want to make this public?

Reply:
Again, why do we turn it back to `public`?

Reply:
This is because you don't put IT and code together, so you have to make it public, can you please add a visible for test tag on it?